server/datastore/mysql/hosts.go

```diff
@@ -5196,6 +5196,18 @@ func numHostsFleetDesktopEnabledDB(ctx context.Context, db sqlx.QueryerContext)
 	return count, nil
 }
 
+func numHostsABMPendingDB(ctx context.Context, db sqlx.QueryerContext) (int, error) {
+	var count int
+	const stmt = `
+		SELECT COUNT(*) FROM fleet.hosts WHERE osquery_host_id IS NULL AND platform NOT IN ('ios', 'ipados', 'android');
```

Comment:
After some thought it seems we cannot use this approach because iOS/iPadOS can be enrolled via ABM too.

Reply:
We can use `host_mdm.enrollment_status`:
```suggestion
SELECT COUNT(*) FROM hosts h LEFT JOIN host_mdm hmdm ON h.id=hmdm.host_id WHERE h.platform IN ('darwin', 'ios', 'ipados') AND hmdm.enrollment_status = 'Pending';
```

Reply:
Looks more accurate.
Thanks.